clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -504,18 +505,73 @@ static Value *EmitISOVolatileStore(CodeGenFunction &CGF, const CallExpr *E) {
   return Store;
 }
 
+static CallInst *CreateBuiltinCallWithAttr(CodeGenFunction &CGF, StringRef Name,
+                                           llvm::Function *FPBuiltinF,
+                                           ArrayRef<Value *> Args,
+                                           unsigned ID) {
+  llvm::CallInst *CI = CGF.Builder.CreateCall(FPBuiltinF, Args);
+  llvm::AttributeList AttrList;
+  // sincos() doesn 't return a value, but it still has a type associated with
+  // it that corresponds to the operand type.
+  if (Name == "sincos")
+    CGF.CGM.getFPAccuracyFuncAttributes(Name, AttrList, ID, Args[0]->getType());
+  else
+    CGF.CGM.getFPAccuracyFuncAttributes(Name, AttrList, ID,
+                                        FPBuiltinF->getReturnType());
```

Comment:
```suggestion
  CGF.CGM.getFPAccuracyFuncAttributes(Name, AttrList, ID, Name == "sincos" ? Args[0]->getType() : FPBuiltinF->getReturnType());
```
reformatted however clang-format wants, but this makes it more clear that the difference is only with the last argument to the call.